### PR TITLE
Update smb.py to fix issue #51

### PR DESCRIPTION
**Pull request by mpgn**
image

### CODE REVIEW

**Comment by Cyb3rC3lt** [comment]
Great job, fixed for the protocols I complained about. I'm not familiar with mmc but that's still an issue it seems.

image

**Comment by Cyb3rC3lt** [comment]
Impersonate module now fixed too

image

**Comment by mpgn** [comment]
Fixed for mmcexec
image

**Comment by mpgn** [comment]
image

**Comment by Cyb3rC3lt** [comment]
I can confirm mmcexec is fixed too now

image

**Comment by mpgn** [comment]
I'm wondering why you have a new line at every line 🤔

**Comment by Cyb3rC3lt** [comment]
Ah yes, looks to be related to me using Terminator as my console. Must look to fix that actually, yours looks better. Here is the result in the default Kali one

image

**Comment by NeffIsBack** [comment]
Hey I am not that fast with reviews 😂 
But glad everything is fixed now 👍

File: cme/protocols/smb/mmcexec.py [hunk 1]
```diff
@@ -219,3 +221,22 @@ def get_output_fileless(self):
...
+            try:
+                self.__smbconnection.getFile(self.__share, self.__output, self.output_callback)
+                break
+            except Exception as e:
+                if str(e).find("STATUS_SHARING_VIOLATION") >= 0:
+                    # Output not finished, let's wait
+                    sleep(2)
+                    pass
+                else:
+                    pass
```
Comment (NeffIsBack) [inline cme/protocols/smb/mmcexec.py:240]: Thats a bit unnecessary but doesn't really matter in the end

**Pull request merged**